resources/autoscaling.rb

```diff
@@ -0,0 +1,169 @@
+property :return_info, [String, Array]
+property :should_decrement_desired_capacity,	[true, false], default: true
+property :asg_name,              String
```

Comment:
The spaces before the type should probably be consistent or just a single space. A single space would be my personal preference so we don't constantly have to update it when we add new properties.

Reply:
done